### DISCUSSION TITLE
BUILD FAILED (Mac OS X 10.12.6 using ruby-build 20211109)

### DISCUSSION THREAD

**Discussion by gpxters** [rbenv/ruby-build#1869] [Build failures]
Very much a newbie trying to install ruby following instructions on AppAcademy to set up an environment.

Trying to run rbenv install 2.5.1 however I am getting the below errors:

Last 10 log lines:
/usr/include/CommonCrypto/CommonRandom.h:35:9: error: unknown type name 'CCCryptorStatus'
typedef CCCryptorStatus CCRNGStatus;
        ^
crypto/rand/rand_unix.c:385:47: error: use of undeclared identifier 'kCCSuccess'
    if (CCRandomGenerateBytes(buf, buflen) == kCCSuccess)
                                              ^
2 errors generated.
make[1]: *** [crypto/rand/rand_unix.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make: *** [all] Error 2

I know this is not much information but could anyone point me in the right direction? I can't find anything online for this specific Mac OS build so at a standstill here.

Thanks for any help.

**Comment by ghost** [comment]
Did you manage to fix it? I have the exact same issue.

**Comment by milongkmt** [comment]
Z
